### PR TITLE
added html css and Script File

### PR DESCRIPTION
**Pull request by nasir-k**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+      `;
+  const newContactlist = document.getElementById("contact-list");
+  newContactlist.append(newContactElement);
+  const deleteBtn = document.getElementById(deleteId);
+  deleteBtn.addEventListener("click", deleteContactElement.bind(null, id));
+  const editBtn = document.getElementById(editId);
+  editBtn.addEventListener("click", editContactHandler);
+};
+
+const closecontactModal = () => {
```
Comment (faran4engg) [inline app.js]: can we use the naming convention properly? In camel case it should be `closeContactModal`

File: app.js [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+  contactModal.classList.remove("visible");
+};
+
+const addcontactModal = () => {
```
Comment (faran4engg) [inline app.js]: same here @nasir-k

File: index.html [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+    <title>Contacts App</title>
+    <link rel="stylesheet" href="style.css" />
+    <script src="app.js" defer></script>
+  </head>
+  <body>
+    <div id="backdrop"></div>
+    <div class="modal card" id="add-modal">
+      <div class="modal__content">
+        <label for="First Name">First Name</label>
+        <input type="text" name="First Name" id="First-Name" />
```
Comment (faran4engg) [inline index.html:25]: I would not give value to the `name` attribute with space !

File: index.html [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+        <label for="Last Name">Last Name</label>
+        <input type="text" name="Last Name" id="Last-Name" />
+        <label for="image-url">Profile Photo(url)</label>
+        <input type="text" name="image-url" id="image-url" />
+        <label for="email">Email</label>
+        <input type="email" name="email" id="" />
+        <label for="phone">Phone No:</label>
+        <input type="number" name="phone" id="" step="10" />
```
Comment (faran4engg) [inline index.html]: what does the step do ?

File: style.css [hunk 5]
```diff
@@ -0,0 +1,312 @@
...
+}
+
+header h1 {
+  margin: 0;
+  color: white;
+  font-size: 1.5rem;
+}
+
+header button {
+  font: inherit;
```
Comment (faran4engg) [inline style.css]: why inherit ?

**Review by faran4engg — CHANGES_REQUESTED** [review]
I have same review comments as Abbas's PR, could you check those and correct please !

**Review by syedabdullahali — APPROVED** [review]

**Pull request merged**